packages/ui/src/views/chatflows/APICodeDialog.js

```diff
@@ -303,7 +309,10 @@ async function query(formData) {
         "${baseURL}/api/v1/prediction/${dialogProps.chatflowid}",
         {
             method: "POST",
-            body: formData
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(formData)
```

Comment:
Dont think this works. Using `formData` , the header should be `'Content-Type': 'application/x-www-form-urlencoded'` not `application/json`, so I prefer to leave this out.

You also cant `JSON stringify` formData

Reply:
oh sorry. you are correct.

thank you for pointing out this point!